Mifare Classic Tool/app/src/main/java/de/syss/MifareClassicTool/Common.java

```diff
@@ -906,18 +903,7 @@ public static int checkMifareClassicSupport(Tag tag, Context context) {
                         // MIFARE Plus EV1 2K/4K SL1
                         return -1;
                     } else {
-                        if ((sak & 1) == 1) { // SAK bit 1 = 1?
-                            // MIFARE Mini
-                            return -1;
-                        } else {
-                            // MIFARE Classic 1k
-                            // MIFARE SmartMX 1k
-                            // MIFARE Plus S 2K SL1
-                            // MIFARE Plus X 2K SL1
-                            // MIFARE Plus SE 1K
-                            // MIFARE Plus EV1 2K/4K SL1
-                            return -1;
-                        }
+                        return -1;
```

Comment:
I did this to make the code more readable and closer to the datasheet linked in an above comment.
I don't like the optimized version. It is not readable.

Reply:
Reverted.